.github/workflows/build-test.yml

```diff
@@ -35,6 +35,47 @@ jobs:
               env:
                 WHITESOURCE_API_KEY: ${{ secrets.WHITESOURCE_API_KEY }}
               run: ./ci/build.sh
+            - name: Upload Build Artifact
+              uses: actions/upload-artifact@v3
+              with:
+                name: artifacts
+                path: artifacts
+
+    test-windows:
+        needs: build
+        name: ${{ matrix.cloud }} JDBC${{ matrix.additionalMavenProfile }} ${{ matrix.category }} / Test on Windows(java ${{ matrix.javaVersion }}, ${{ matrix.cloud }} )
```

Comment:
Maybe let's rename thew builds for windows to: 
```
${{ matrix.cloud }} Windows java ${{ matrix.javaVersion }} JDBC${{ matrix.additionalMavenProfile }} ${{ matrix.category }}
```

then we can also adjust other build names

Reply:
Updated names for Mac, Windows and Linux using centos images.